### PR TITLE
reCaptcha check

### PR DESCRIPTION
**Pull request by battye**
Check to make sure the RECAPTCHA_LANG value is in the list specified at https://developers.google.com/recaptcha/docs/language

See: https://github.com/phpbb/phpbb-translation-validator/issues/49

### CODE REVIEW

File: src/Phpbb/TranslationValidator/Validator/FileValidator.php [hunk 1]
```diff
@@ -39,6 +39,81 @@ class FileValidator
...
+		'sv',
+		'ta',
+		'te',
+		'th',
+		'tr',
+		'uk',
+		'ur',
+		'vi',
+		'zu',
+		'', // Allow empty strings
```
Comment (Crizz0) [inline src/Phpbb/TranslationValidator/Validator/FileValidator.php]: Why? Without this value, the recaptcha should not work. We ask for a default "en", when nothing fits:

> If no code exists for your language you can use "en" or leave the string empty

Comment (battye) [reply src/Phpbb/TranslationValidator/Validator/FileValidator.php]: If no code exists for their language they can use en or leave the string empty. So by having the `''` as part of the array it means validation will still pass if the translator chooses to leave the string empty.

Comment (Crizz0) [reply src/Phpbb/TranslationValidator/Validator/FileValidator.php]: Good.

**Comment by Crizz0** [comment]
Works good, but I'm not happy with the WARNING output, when you delect a lang, which is not available, like "dex". 

It should be FATAL or at least an ERROR. I think an instant deny it to much, but an ERROR should do it. What do you think?

**Comment by battye** [comment]
I think sounds reasonable too, I'll change it to ERROR now.

**Pull request merged**